### ISSUE TITLE
`kvm: no hardware support for kvm_intel` on Zen 3 CPU

### ISSUE THREAD

**Issue by KaeLL** [clearlinux/distribution#2780]
It should still work, right? Probably not officially supported, so I just want to know how to make it work.

**Comment by fenrus75** [comment]
you won;t get intel-kvm support... but you should get amd-kvm support
(assuming it's enabled in the bios etc)

**Comment by marioroy** [comment]
Kernel configuration:

/etc/kernel/cmdline-removal.d/kvm-intel.conf
```text
kvm-intel.nested=1
```

/etc/kernel/cmdline.d/kvm-amd.conf
```text
kvm-amd.nested=1
```

Update boot configuration:
```text
sudo clr-boot-manager update
```

**Comment by KaeLL** [comment]
@fenrus75 AMD SVM is enabled, still doesn't work.
@marioroy I can't even boot the liveusb. Is there still a way to change the liveusb kernel parameters?

**Comment by KaeLL** [comment]
Apparently there is a way to change the live img kernel parameters, which I tried, but still to no avail, and I still get the same error. Here is the original string:
`initrd=\EFI\org.clearlinux\freestanding-00-early-ucode.cpio initrd=\EFI\org.clearlinux\freestanding-i915-firmware.cpio.zstd initrd=\EFI\BOOT\initrd.gz quiet console=tty0 console=ttyS0,115200n8 cryptomgr.notests init=/usr/bin/initra-desktop initcall_debug intel_iommu=igfx_off kvm-intel.nested=1 no_timer_check noreplace-smp page_alloc.shuffle=1 rcupdate.rcu_expedited=1 rootfstype=ext4,btrfs,xfs,f2fs tsc=reliable rw rootwait`

Neither removing nor setting `kvm-intel.nested=0`, regardless of whether `kvm-amd.nested=1` is present or not, works.

**Comment by marioroy** [comment]
What are you looking at to determine kvm not working? It requires SVM to be enabled in the bios. Notice the first message from dmesg stating no hardware support. Ignore that line early in the boot process. KVM is working on my AMD box. I run QEMU virtual machines using KVM.
```text
$ sudo dmesg | grep kvm
[    0.494324] kvm: no hardware support
[    5.333607] kvm: Nested Virtualization enabled
[    5.333609] SVM: kvm: Nested Paging enabled

$ lsmod | grep kvm
kvm_amd               135168  0
ccp                   106496  1 kvm_amd
```

**Comment by marioroy** [comment]
I disabled SVM in the bios simply to see what dmesg looks like and whether able to run QEMU virtual machines requesting KVM via the  -enable-kvm command-line argument.
```text
$ sudo dmesg | grep kvm
[    0.490594] kvm: no hardware support
[    4.269407] kvm: disabled by bios
[    4.356562] kvm: disabled by bios
[    4.375808] kvm: disabled by bios
[    4.394387] kvm: disabled by bios
[    4.413263] kvm: disabled by bios
[    4.431165] kvm: disabled by bios
[    4.449025] kvm: disabled by bios
[    4.465492] kvm: disabled by bios
[    4.481023] kvm: disabled by bios
[    4.499782] kvm: disabled by bios

$ lsmod | grep kvm

$ bash windows-11/windows-11.sh
Could not access KVM kernel module: No such file or directory
qemu-system-x86_64: failed to initialize kvm: No such file or directory
```

**Comment by KaeLL** [comment]
@marioroy 
> What are you looking at to determine kvm not working?

What I'm trying to do is install CL on my new AMD machine. I'm not messing with Qemu/KVM in any way.

But when booting from the USB flash with CL img on it, the only thing I get is the message in the title.
Here's to show SVM is enabled
![svm_bios](https://user-images.githubusercontent.com/4423144/205730885-21def698-a6b4-4ebd-9d42-e65e953f889e.jpg)
Following what you said, I tried editing the kernel parameters when booting the liveusb img
![IMG_20221205_165131_140](https://user-images.githubusercontent.com/4423144/205731601-ca85c528-3317-48e2-8852-06d573d455ae.jpeg)

but it doesn't work and I still get the same message every single time and it stays like that until I reboot the machine
![IMG_20221205_165155_346](https://user-images.githubusercontent.com/4423144/205731706-e5deb084-f012-4ef1-9548-47216a279cc0.jpeg)

**Comment by KaeLL** [comment]
It's been pointed out to me that this may be only the kernel reporting something that may be completely unrelated to the issue. If so, then I have no idea about what to do next.

**Comment by bryteise** [comment]
Try removing quiet from your kernel command line and maybe there will be something else that shows up.

**Comment by KaeLL** [comment]
@bryteise I did, and turns out that some update broke things along the way and 37010 won't boot. I suspect the same goes for quite a lot of the newer versions. I even considered doing something like a manual bisect to find out which one broke, but it'd take a long time since I'm on a 30Mb/s internet connection right now, so I gave up on it.
I then tried 36010 (the "official" current version on CL's website), which booted, but the main screen looks weird with a yellow and red-ish tone, perhaps due to drivers?
![WhatsApp Image 2022-12-06 at 13 32 41](https://user-images.githubusercontent.com/4423144/205974748-2911453f-fc9b-4e14-9bb8-e0249ee32eb0.jpg)

Screenshots look normal, though
![Wv1GZpO](https://user-images.githubusercontent.com/4423144/205975610-d4431a2b-6c9d-494e-8216-5a543237c626.png)
